ros2_batch_job/__main__.py

```diff
@@ -395,6 +398,12 @@ def run(args, build_function, blacklisted_package_names=None):
             log("@{bf}==>@| Contents of `ros2.repos`:")
             with open('ros2.repos', 'r') as f:
                 print(f.read())
+            # Download and merge supplemental repos file
+            if args.supplemental_repo_file_url is not None:
+                job.run(['curl', '-sk', args.supplemental_repo_file_url, '-o', 'supplemental.repos'])
+                log("@{bf}==>@| Contents of `supplemental.repos`:")
+                with open('supplemental.repos', 'r') as f:
+                    print(f.read())
```

Comment:
Since these four lines are identical to the ones above I would recommend moving them to a function.